app/helpers/user_list_upload/user_save_attempts_helper.rb

```diff
@@ -31,6 +31,9 @@ def user_save_status_text(status)
   end
 
   def user_save_background_color_for_attribute(user_row, attribute)
+    # Skip uniqueness validations for display purposes to avoid N+1 queries
+    user_row.user.skip_uniqueness_validations = true
```

Comment:
Je ne pense pas qu'ici dans le helper qui gère la vue soit le bon endroit pour faire ça :).
À mon avis tu peux le faire dans la méthode `user_row#user`, au même titre que `require_title_presence`

Reply:
c'est fait dans le dernier commit mais je m'interroge. Est ce que ce n'est pas risqué de le faire dans la méthode `user_row#user` ? On ne risque pas d'avoir des doublons si on fait des changements dans l'app à l'avenir avec cette ligne qui serait difficile à débug ?
Il vaudrait mieux pas passer par une autre méthode dans le model par exemple :

```ruby
  def user_for_display
    # Skip uniqueness validations for display purposes to avoid N+1 queries
    user.tap { |u| u.skip_uniqueness_validations = true }
  end
  ```